pkg/handler/custom_token.go

```diff
@@ -73,11 +87,23 @@ func (h *Handler) HandlerGuildCustomTokenConfig(c *gin.Context) {
 
 	req.Id = token
 
-	// add to custom token config
-	if err := h.entities.CreateGuildCustomTokenConfig(req); err != nil {
-		c.JSON(http.StatusInternalServerError, gin.H{"error": "cannot add to token config"})
+	// check token config exists or not
+	checkExistTokenConfig, err := h.entities.CheckExistTokenConfig(req.Id, req.GuildID)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	if !checkExistTokenConfig {
```

Comment:
if checkExistTokenConfig: c.JSON(http.StatusBadRequest, gin.H{"message": "Your guild has already added this token."})
the success case should be at the end of handler.